projects/gopacket/Dockerfile

```diff
@@ -15,7 +15,7 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder-go
-RUN go get github.com/google/gopacket
+RUN git clone https://github.com/google/gopacket.git
```

Comment:
For at least this and many others, we need to set WORKDIR below to `$SRC/gopacket`. 

Otherwise this fails to build:

```
-: no required module provides package github.com/google/gopacket/layers; to add it:                                                                                                                                                                                                      
        go get github.com/google/gopacket/layers     
```

Reply:
Done.